tests/integration/courses.test.js

```diff
@@ -1121,8 +1121,15 @@ describe('COURSES ROUTES - PUT /courses/{_id}', () => {
 
       expect(response.statusCode).toBe(200);
 
-      const course = await Course.countDocuments({ _id: coursesList[17]._id, ...payload }).lean();
-      expect(course).toEqual(1);
+      const courseUpdated = await Course.countDocuments({ _id: coursesList[17]._id, ...payload });
+      expect(courseUpdated).toEqual(1);
+
+      const historyCreated = await CourseHistory.countDocuments({
+        course: coursesList[17]._id,
+        action: ESTIMATED_START_DATE_EDITION,
+        update: { estimatedStartDate: { from: '', to: '2022-05-31T08:00:00.000Z' } },
+      });
+      expect(historyCreated).toEqual(1);
```

Comment:
Que penses-tu de faire ces tests dans le premier test ? (`should update course`)

Reply:
je ne comprend pas ton retour, des lignes 1105 a 1133 on est dans le test `should update course`.

Reply:
Excuse moi j'ai mal lu les tests ^^'